ncl/ncl_entries/days_in_month.ipynb

```diff
@@ -0,0 +1,194 @@
+     ]
+    }
+   ],
+   "source": [
+    "from calendar import monthrange\n",
+    "\n",
+    "month = 2\n",
+    "year = 1500\n",
+    "first_day, num_days = monthrange(year, month)\n",
+    "print(f\"Month {month} in {year} has {num_days} days\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "82c5bb43f75c474e",
+   "metadata": {},
+   "source": [
+    "However, the `calendar` only uses a proleptic Gregorian calendar. If you need to work with other calendars, the `calendar` module will not be sufficient."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "7bfc78baeee6cef4",
+   "metadata": {},
+   "source": [
+    "## Using `cftime`\n",
+    "[`cftime`](https://unidata.github.io/cftime/index.html) supports all of the [CF Convention calendars](https://cfconventions.org/cf-conventions/cf-conventions#calendar). Here's how you can use `cftime` to calculate the number of days in a month."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "id": "f762cbca2cf59dfa",
+   "metadata": {
+    "ExecuteTime": {
+     "end_time": "2024-06-02T23:36:55.219575Z",
+     "start_time": "2024-06-02T23:36:55.126580Z"
+    }
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "standard calendar: \t\t\t\t month 2 in 1500 has 29 days\n",
+      "proleptic Gregorian calendar: \t month 2 in 1500 has 28 days\n",
+      "Julian calendar: \t\t\t\t month 2 in 1500 has 29 days\n",
+      "360 day calendar: \t\t\t\t month 2 in 1500 has 30 days\n"
+     ]
+    }
+   ],
+   "source": [
+    "import cftime\n",
+    "\n",
+    "day = 1\n",
+    "month = 2\n",
+    "year = 1500\n",
+    "\n",
+    "days = cftime.datetime(year, month, day, calendar='standard').daysinmonth\n",
+    "print(f\"standard calendar: \\t\\t\\t\\t month {month} in {year} has {days} days\")\n",
+    "\n",
+    "days = cftime.datetime(year, month, day, calendar='proleptic_gregorian').daysinmonth\n",
+    "print(f\"proleptic Gregorian calendar: \\t month {month} in {year} has {days} days\")\n",
+    "\n",
+    "days = cftime.datetime(year, month, day, calendar='julian').daysinmonth\n",
+    "print(f\"Julian calendar: \\t\\t\\t\\t month {month} in {year} has {days} days\")\n",
+    "\n",
+    "days = cftime.datetime(year, month, day, calendar='360_day').daysinmonth\n",
+    "print(f\"360 day calendar: \\t\\t\\t\\t month {month} in {year} has {days} days\")"
+   ]
+  },
+  {
+   "metadata": {},
+   "cell_type": "markdown",
+   "source": [
+    "## Differences between `cftime` and NCL's `days_in_month`\n",
+    "\n",
+    "### Calendars\n",
+    "The calendars available in `cftime` and NCL's `days_in_month`, while both based on the CF Conventions calendars, are slightly different.\n",
+    "\n",
+    "- `cftime` does support using `\"none\"` as a calendar type, but not for this particular function. \n",
+    "- NCL's `days_in_month` does not support the `proleptic_gregorian` calendar.\n",
```

Comment:
Is it helpful to comment on these types of differences?  It seems easy in here, but just thinking about it moving forward.

Reply:
I was thinking it was super important, but maybe that's a dev vs user bias. 

I think in the future, it would be reasonable to cut and collect this info into a dedicated, like, calendars entry that we could link to instead of potentially repeating information, but I don't think we have enough content (or full enough knowledge of the NCL calendars) to do that yet.

Reply:
Ok, yeah - I guess that's more where I was going with this (though my comment def did not indicate that).  It feels a little detailed for a single function entry, but at a higher level I totally agree it's helpful.